nuget/MyGet.ps1

```diff
@@ -0,0 +1,9 @@
+$nuget = $env:NuGet
+
+#parse the version number out of the Jekyll _config.yml
+$yaml = (Select-String $env:SourcesPath\_config.yml -pattern "current_version").ToString().Split(" ");
```

Comment:
Yes, this is a bit cleaner: 
`$bsversion = ((Get-Content ..\package.json) -join "`n" | ConvertFrom-Json).version`